main.py

```diff
@@ -2387,14 +2387,14 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                             messageForHandUpgrade(tags['user-id'], tags['display-name'], self, channel, isWhisper)
 
                         droplink = config["siteHost"] + "/booster?user=" + sender
-                        self.message(channel, "{user}, you open a {type} booster pack and you get: {droplink}".format(
+                        self.message(channel, "{user}, you open a {type} booster: {droplink}".format(
                             user=tags['display-name'], type=packname, droplink=droplink), isWhisper=isWhisper)
                     except InvalidBoosterException:
                         self.message(channel, "Invalid booster type. Packs available right now: %s." % visiblepacks,
                                      isWhisper=isWhisper)
                     except CantAffordBoosterException as exc:
                         self.message(channel,
-                                     "{user}, sorry, you don't have enough points to buy a {name} booster pack. You need {points}.".format(
+                                     "{user}, sorry, you don't have enough points to buy a {name} booster. You need {points}.".format(
```

Comment:
Idea for further shortening: `{name} pack` would save three characters over `{name} booster}`